### PR TITLE
Created new Command Menu images for 4 articles

### PR DESCRIPTION
**Pull request by captainbrosset**
One of the image used to show what the command menu looked like was shared between 4 articles.

This was a problem because it showed content that was only correct for 1 of those 4 articles.

In this commit, I made sure the image was only used in the correct article.
And I created new images for the 3 other articles.

I also removed the :::image::: extension usage and replaced it with standard markdown image tagging.

And I also removed a couple images that, I think, weren't that important. The instructions were clear enough (or at least I tried to make them clearer where possible) to avoid having too many screenshots.

* emulate-vision-deficiencies: [before](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies)/[after](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies?branch=pr-en-us-1626)
* reduced-motion-simulation: [before](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/reduced-motion-simulation)/[after](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/reduced-motion-simulation?branch=pr-en-us-1626)
* print-preview: [before](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/print-preview)/[after](https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/print-preview?branch=pr-en-us-1626)

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md [hunk 1]
```diff
@@ -24,43 +24,31 @@ To better meet the needs of your users with [color vision deficiency](https://ww
...
+To simulate how a person with a vision deficiency would see your web page, open the [Rendering Tools](../rendering-tools/index.md).
 
-1.  To open the Rendering tool, select the `...` menu item in the toolbar.
-1.  Select **More tools**.
-1.  Select **Rendering**.
+1.  Click **Customize and control DevTools** (`...`) in the toolbar.
+1.  Go to **More tools** > **Rendering** to open the **Rendering** panel in the drawer.
 
-    :::image type="content" source="../media/getting-to-the-rendering-tools.msft.png" alt-text="Opening the Rendering tool." lightbox="../media/getting-to-the-rendering-tools.msft.png":::
+    ![Opening the Rendering panel from the More tools menu](../media/getting-to-the-rendering-tools.msft.png)
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md:32]: If you make this image narrower than 1200px, by dragging the window to be smaller before capturing, the result will be crisper.  There's too much info in this screenshot, so the result is small, blurry UI elements.  I tried implementing this, but the borders weren't matching the other images, on my machine.

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md:32]: try per DCG:
```suggestion
> [!div class="mx-imgBorder"]
> ![Opening the Rendering panel from the More tools menu](../media/getting-to-the-rendering-tools.msft.png)
```

File: microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md [hunk 2]
```diff
@@ -24,43 +24,32 @@ To better meet the needs of your users with [color vision deficiency](https://ww
...
 
-1.  To open the Rendering tool, select the `...` menu item in the toolbar.
-1.  Select **More tools**.
-1.  Select **Rendering**.
+1.  Click **Customize and control DevTools** (`...`) in the toolbar.
+1.  Go to **More tools** > **Rendering** to open the **Rendering** panel in the drawer.
 
-    :::image type="content" source="../media/getting-to-the-rendering-tools.msft.png" alt-text="Opening the Rendering tool." lightbox="../media/getting-to-the-rendering-tools.msft.png":::
+    > [!div class="mx-imgBorder"]
+    > ![Opening the Rendering panel from the More tools menu](../media/getting-to-the-rendering-tools.msft.png)
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md:33]: If you make this image narrower than 1200px, by dragging the window to be smaller before capturing, the result will be crisper. There's too much info in this screenshot, so the result is small, blurry UI elements. I tried implementing this, but the borders weren't matching the other images, on my machine.

File: microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md [hunk 3]
```diff
@@ -24,43 +24,32 @@ To better meet the needs of your users with [color vision deficiency](https://ww
...
+To simulate how a person with a vision deficiency would see your web page, open the [Rendering Tools](../rendering-tools/index.md).
 
-1.  To open the Rendering tool, select the `...` menu item in the toolbar.
-1.  Select **More tools**.
-1.  Select **Rendering**.
+1.  Click **Customize and control DevTools** (`...`) in the toolbar.
+1.  Go to **More tools** > **Rendering** to open the **Rendering** panel in the drawer.
 
-    :::image type="content" source="../media/getting-to-the-rendering-tools.msft.png" alt-text="Opening the Rendering tool." lightbox="../media/getting-to-the-rendering-tools.msft.png":::
+    > [!div class="mx-imgBorder"]
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md:32]: In Preview in VS Code, removing > caused the preview to be truncated here.  So, > seems required (probably).  I haven't tested that in the built/rendered docs, only in VS Code locally.

You could remove this tagging, or apply it to other images in this PR.

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md (hunk 1 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md]: No I don't think `>` is required.

File: microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md (hunk 2 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/accessibility/emulate-vision-deficiencies.md:33]: I took another screenshot. Let me know what you think.

**Pull request merged**